pandas/core/strings.py

```diff
@@ -726,59 +763,35 @@ def f(x):
     return _na_map(f, arr)
 
 
-def str_strip(arr, to_strip=None):
-    """
-    Strip whitespace (including newlines) from each string in the array
-
-    Parameters
-    ----------
-    to_strip : str or unicode
-
-    Returns
-    -------
-    stripped : array
-    """
-    return _na_map(lambda x: x.strip(to_strip), arr)
-
-
-def str_lstrip(arr, to_strip=None):
-    """
-    Strip whitespace (including newlines) from left side of each string in the
-    array
-
-    Parameters
-    ----------
-    to_strip : str or unicode
-
-    Returns
-    -------
-    stripped : array
-    """
-    return _na_map(lambda x: x.lstrip(to_strip), arr)
-
-
-def str_rstrip(arr, to_strip=None):
+def str_strip(arr, to_strip=None, side='both'):
     """
-    Strip whitespace (including newlines) from right side of each string in the
-    array
+    Strip whitespace (including newlines) from each string in the ``Series``/``Index``.
 
     Parameters
     ----------
     to_strip : str or unicode
+    side : {'left', 'right', 'both'}, default 'both'
 
     Returns
     -------
-    stripped : array
+    stripped : Series/Index (object)
     """
-    return _na_map(lambda x: x.rstrip(to_strip), arr)
+    if side == 'both':
+        f = lambda x: x.strip(to_strip)
+    elif side == 'left':
+        f = lambda x: x.lstrip(to_strip)
+    elif side == 'right':
+        f = lambda x: x.rstrip(to_strip)
+    else:  # pragma: no cover
+        raise ValueError('Invalid side')
+    return _na_map(f, arr)
 
 
 def str_wrap(arr, width, **kwargs):
-    r"""
-    Wrap long strings to be formatted in paragraphs.
-
-    This method has the same keyword parameters and defaults as
-    :class:`textwrap.TextWrapper`.
+    """
+    Wrap long strings in the ``Series``/``Index`` to be formatted in paragraphs with
+    length less than a given width.
+    This method has the same keyword parameters and defaults as :class:`textwrap.TextWrapper`.
```

Comment:
Can you leave a blank line between the two sentences? (sphinx ignores newlines, unless separated by blank lines)